emote/memory/table.py

```diff
@@ -14,6 +21,24 @@
 from .strategy import EjectionStrategy, SampleStrategy
 
 
+logger = logging.getLogger(__name__)
+
+
+class TableSerializationVersion(enum.Enum):
+    """The version of the memory serialization format."""
+
+    Legacy = 0
```

Comment:
Legacy wasn't versioned, but to avoid making "unversioned" = -1 I went for `unversioned = 0`. Does that make sense? So V1 is the first actually versioned protocol.